.github/workflows/configs/.markdownlint.yaml

```
@@ -0,0 +1,132 @@
+# Original file source: https://github.com/MicrosoftDocs/PowerShell-Docs/blob/2472d5d3ffd90862d11d2d6dfeb9faa1fba7523c/.markdownlint.yaml
+# Docs: https://github.com/DavidAnson/markdownlint?tab=readme-ov-file
+---
```

actionlint (1.7.7)

"on" section is missing in workflow

(syntax-check)